projects/containernetworking/plugins/build/create_binaries.sh

```diff
@@ -37,7 +37,7 @@ function build::plugins::binaries(){
   mkdir -p "$BIN_PATH"
   git clone "$CLONE_URL" "$REPO"
   cd "$REPO"
-  git checkout "$TAG"
+  git switch -c "$TAG"
```

Comment:
'switch -c' will not really checkout that tag. It will just create another branch with the tag name.
Can we use,
git checkout -b "$TAG" "$TAG"

Reply:
I changed it to `git switch -c "$TAG" "$TAG"` which is the same thing.